packages/patterns-react/src/patterns/CardWithImages/CardsWithImages.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import root from 'window-or-global';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { DDS_CARDS_WITH_IMAGES } from '../../internal/FeatureFlags';
+import CardsWithImagesGroup from './CardsWithImagesGroup';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Cards with images pattern
+ *
+ * @param {object} props props object
+ * @param {Array} props.cardsGroup cardsGroup array with title, groupCard and cards properties
+ * @returns {object} JSX Object
+ */
+const CardsWithImages = ({ theme, cardsGroup }) => {
+  useLayoutEffect(() => {
+    root.addEventListener(
+      'resize',
+      _matchHeight(`.${prefix}--cards-with-images-group__cards`)
+    );
+  });
+
+  /**
+   * Match heights of child elements
+   *
+   * @private
+   * @param {*} parentEl Parent element selector
+   */
+  const _matchHeight = parentEl => {
+    const parentEls = document.querySelectorAll(parentEl);
+    parentEls.forEach(parentEl => {
+      const childEls = parentEl.childNodes;
+      childEls.forEach(el => {
+        el.style.height = null;
+      });
+      const maxHeight = _getMaxHeight(childEls);
+      childEls.forEach(el => {
+        el.style.height = maxHeight + 'px';
+      });
+    });
+  };
```

Comment:
@RaphaelAmadeu @sannminwin @kennylam @annawen1 @emerson-pereira @andysherman2121 @carthick @photodow I feel like this has been created a number of times. Should we move this into `utilities`?

Reply:
I think getting the hightest div, making all div equal height etc .. are very common scenario in frontend work and we should get this tool handy in `utilities`.

Reply:
Agreed since it's being used in multiple places

Reply:
@mkothur can you move this into `packages/utilities`?

Reply:
I was just talking to @RaphaelAmadeu about this the other day. Yeah, I 100% agree that this needs to be in a utility. I can create an issue for it: https://github.com/carbon-design-system/ibm-dotcom-library/issues/816

Reply:
One other thing to note on this is I'm seeing we're doing a lot of resizing of the entire card itself. This is causing some improper alignment in the card's content area.

![image](https://user-images.githubusercontent.com/3793636/70641953-09ea0500-1c04-11ea-8e52-2cd7cde157ae.png)

If instead, we match the height of all the titles and body text within the card everything should align to what the specs are asking for along with the same height of the cards themselves.

![image](https://user-images.githubusercontent.com/3793636/70642385-bcba6300-1c04-11ea-9202-eb96b41921f8.png)

Reply:
I guess the alignment issue can be solved with flex. If we make the utility this specific it would  be limited by the patterns.